pkg/platform/model/recipe.go

```diff
@@ -133,17 +138,20 @@ func commitToOrder(commitID strfmt.UUID, owner, project string, hostPlatform *st
 	return order, nil
 }
 
-func fetchRecipeID(commitID strfmt.UUID, owner, project string, hostPlatform *string) (*strfmt.UUID, *failures.Failure) {
+func fetchRecipeID(commitID strfmt.UUID, owner, project, orgID string, private bool, hostPlatform *string) (*strfmt.UUID, *failures.Failure) {
 	var err error
 	params := iop.NewSolveOrderParams()
 	params.Order, err = commitToOrder(commitID, owner, project, hostPlatform)
 	if err != nil {
 		return nil, FailOrderRecipes.Wrap(err)
 	}
+	if private {
+		params.OrganizationID = &orgID
+	}
```

Comment:
Any reason why we shouldn't just always set this?

Reply:
If this is set on public projects we do not get the correct recipeID back